### PR TITLE
Type conversion

### PR DESCRIPTION
**Pull request by creek0810**
add type conversion operation
- toString
- toInt

### CODE REVIEW

File: tests/test__collection_api.py [hunk 1]
```diff
@@ -4759,3 +4759,79 @@ def test__insert_do_not_modify_input(self):
             fetched_document, stored_document,
             msg='Modifying the found document afterwards does not modify the stored document.')
         self.assertEqual(dict(original_document, date=None), dict(stored_document, date=None))
+
+    @skipIf(not _HAVE_PYMONGO, 'pymongo not installed')
+    def test__aggregate_to_string(self):
+        collection = self.db.collection
+        collection.insert_many([
```
Comment (pcorpet) [inline tests/test__collection_api.py]: insert_one should be enough here

File: tests/test__collection_api.py [hunk 2]
```diff
@@ -4759,3 +4759,79 @@ def test__insert_do_not_modify_input(self):
...
+            'integer': '100',
+            'date': '2018-03-27T00:58:51.538Z',
+            'none': None
+        }]
+        self.assertEqual(expect, list(actual))
+
+    @skipIf(not _HAVE_PYMONGO, 'pymongo not installed')
+    def test__aggregate_to_int(self):
+        collection = self.db.collection
+        collection.insert_many([
```
Comment (pcorpet) [inline tests/test__collection_api.py]: Same here

File: tests/test__collection_api.py [hunk 3]
```diff
@@ -4759,3 +4759,79 @@ def test__insert_do_not_modify_input(self):
...
+            {
+                'boolean_true': True,
+                'boolean_false': False,
+                'integer': 100,
+                'double': 1.999,
+                'decimal': decimal128.Decimal128('5.5000')
+            }
+        ])
+        actual = collection.aggregate(
```
Comment (pcorpet) [inline tests/test__collection_api.py:4808]: Same here

**Review by pcorpet — CHANGES_REQUESTED** [review]
Thanks for your work. Please also add a test in test__mongomock to compare with pymongo behaviour.

File: tests/test__collection_api.py (hunk 1 above)
Comment (creek0810) [reply tests/test__collection_api.py]: Done

**Comment by creek0810** [comment]
I have added test in test__mongomock and changed insert_many to insert_one.

**Review by pcorpet — APPROVED** [review]

**Pull request merged**